R/cf.R

```diff
@@ -585,6 +606,9 @@ add.cf <- function(cf1, cf2, a = 1.0, b = 1.0) {
 
 #' Arithmetically divide correlators
```

Comment:
Division does not make too much sense anyway, so I'm not too worried that what is done below is clearly not correct. For the cases where the signal of interest is either in the real or imaginary part, it would do the correct thing.